### PR TITLE
Add batch actions (print/select all) top and bottom

### PR DESCRIPTION
**Pull request by teneightfive**
This follows up on a conversation had around "[As a monies clerk I'd like the batch view to be printable, because I have to print](https://www.pivotaltracker.com/story/show/99023134)" to include the print button at the top and bottom of the transaction list.

### CODE REVIEW

File: mtp_cashbook/templates/cashbook/includes/table_actions.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<div class="batch-actions">
+  <p class="print">
+    <a href="#print-dialog" class="js-Dialog">{% trans "Print these transactions" %}</a>
+  </p>
+  <p class="select-all">
+    <input type="checkbox" id="select-all-{{ location }}" class="Checkbox--right js-SelectAll" data-name="{{ field_name }}">
+    <label for="select-all-{{ location }}">{% trans "Select all" %}</label>
+  </p>
+</div>
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/includes/table_actions.html]: Can we add this via JS, as it doesn't work without?

File: mtp_cashbook/templates/cashbook/transaction_batch_list.html [hunk 2]
```diff
@@ -55,19 +57,9 @@
...
-          </th>
-          <th colspan="2" class="select-all">
-            <input type="checkbox" id="select-all" class="Checkbox--right">
-            <label for="select-all">{% trans "Select all" %}</label>
-          </th>
-        </tr>
-      </tfoot>
     </table>
+
+    {% include "cashbook/includes/table_actions.html" with location='footer' field_name=form.transactions.html_name %}
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Why not put these in `thead` and `tfoot`? Strikes me that the select all, at least, would make more sense (and work better for users of assistive tech?) in those…

Comment (teneightfive) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Yep, they are linked to the contents of the table so would make more sense within `thead` and `tfoot`.

File: mtp_cashbook/templates/cashbook/includes/table_actions.html (hunk 1 above)
Comment (teneightfive) [inline mtp_cashbook/templates/cashbook/includes/table_actions.html]: As discussed yesterday, I'm not about this right now. 

Without JS they are currently hidden from the view and moving them to JS would either mean having the markup in JS files or adding JS templating, which might increase the complexity around internationalization. 

I'm happy either way but at the moment leaning towards what we have and not involving JS templates as it keeps all the i18n handled by django. 

What's your preference?

Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/includes/table_actions.html]: Leaving as-is works for me, just so long as we've considered our reasons for one choice or the other!

**Comment by teneightfive** [comment]
fixup! Moved the batch actions into `thead` and `tfoot` and refactored some of the sass to compensate.

**Comment by teneightfive** [comment]
@SteveMarshall okay to rebase and then merge?

**Pull request merged**